pkgs/development/libraries/gstreamer/base/default.nix

```diff
@@ -80,7 +80,7 @@ stdenv.mkDerivation rec {
     ++ lib.optional (!stdenv.isDarwin) libvisual
     ++ lib.optional enableAlsa alsaLib
     ++ lib.optionals enableX11 [ libXv pango ]
-    ++ lib.optional enableWayland wayland
+    ++ lib.optionals enableWayland [ wayland wayland-protocols ]
```

Comment:
Is perhaps now needing wayland-protocols symptomatic of an underlying issue that we should fix instead?

Reply:
Yeah, this looks like it should be fixed at some other point, although I cannot see at first glance what might be causing it.